ios/FluentUI/Generated/FluentUIStyle.generated.swift

```diff
@@ -0,0 +1,2958 @@
+/// Autogenerated file
```

Comment:
If this is generated, should it be checked into source?

Reply:
This is because the file is not auto-generated as part of the FluentUI Build process.
This file won't be changing frequently unless a new control is added or a change is made on the tokens from the design team.

But any time it's changed, it should be re-generated from the YAML token files and checked in here since all the controls depend on it.